core/services/flux_monitor.go

```diff
@@ -157,7 +157,9 @@ func connectCheckers(ctx context.Context, jobMap map[string][]DeviationChecker,
 			// XXX: Add mechanism to asynchronously communicate when a job spec has
 			// an ethereum interaction error.
 			// https://www.pivotaltracker.com/story/show/170349568
-			logger.ErrorIf(connectSingleChecker(ctx, checker, client))
+			err := checker.Connect(client)
+			logger.ErrorIf(err)
```

Comment:
Does this panic if err? Seems like execution might continue here, which might be undesirable.

Reply:
No this just logs an error, it does not panic